### ISSUE TITLE
make use of `os.get_terminal_size()`

### ISSUE THREAD

**Issue by pfuntner** [pfuntner/toys#99]
I've only just today learned about the [`os.get_terminal_size()`](https://docs.python.org/3/library/os.html#os.get_terminal_size) method.  Some of my tools need to know the screen geometry and I think I created a method in my BrunoUtils class but it uses a homegrown approach to get the same information:

```
  @classmethod
  def cols(cls):
    """
      This is designed to return the number of columns in the user's display.
      Return value: The number of columns as an integer.
    """

    return cls.extract('stty size < /dev/tty', '\d+\s+(\d+)', 80)

  @classmethod
  def rows(cls):
    """
      This is designed to return the number of rows in the user's display.
      Return value: The number of rows as an integer.
    """

    return cls.extract('stty size < /dev/tty', '(\d+)\s+\d+', 24)
```

I think using `os.get_terminal_size()` makes more sense and I can change the method in BrunoUtils.

I also remember this was not the first method I used - I think at one time I relied on the `tput` command but I think that wouldn't always work.

**Comment by pfuntner** [comment]
Actually, I was having trouble getting `os.terminal_size()` to work:

```
[mrbruno@bruno-meerkat 08:34:08 toys master]$ peval -i os -i sys 'os.terminal_size()'
Traceback (most recent call last):
  File "/home/mrbruno/bin/peval", line 30, in <module>
    print(eval(expression))
  File "<string>", line 1, in <module>
TypeError: structseq() missing required argument 'sequence' (pos 1)
$ peval -i os -i sys 'os.terminal_size(sys.stdout)'
Traceback (most recent call last):
  File "/home/mrbruno/bin/peval", line 30, in <module>
    print(eval(expression))
  File "<string>", line 1, in <module>
io.UnsupportedOperation: not readable
$ peval -i os -i sys 'os.terminal_size(sys.stdout.fileno)'
Traceback (most recent call last):
  File "/home/mrbruno/bin/peval", line 30, in <module>
    print(eval(expression))
  File "<string>", line 1, in <module>
TypeError: constructor requires a sequence
$
```
🤷 I have no idea how this is supposed to work!!

The doc says:

> _[shutil.get_terminal_size()](https://docs.python.org/3/library/shutil.html#shutil.get_terminal_size) is the high-level function which should normally be used, os.get_terminal_size is the low-level implementation._

Which seems much easier and _actually works_!

```
$ peval -i shutil 'shutil.get_terminal_size()'
os.terminal_size(columns=168, lines=45)
$
```

**Comment by pfuntner** [comment]
Addressed in 367c69b... plus more housekeeping in BrunoUtils.py!
